### PR TITLE
Robins install swift changes

### PR DESCRIPTION
**Pull request by RobinBateman808**
### Motivation:

Help developers get started using Swift.

### Modifications:

Lightly edited "getting started > installing swift" 

### Result:

Help new users get started using swift from an instructional vs. functional approach.

### CODE REVIEW

**Comment by RobinBateman808** [comment]
@federicobucchi - these are very light changes to the page. Plmk if you have questions. Thanks!

File: _includes/getting-started/_installing.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ## Installing Swift
 
-If you don't have Swift installed, [install it first](/install).
+To kickstart your journey, [install Swift](/install) to begin using it on **macOS**, **Linux**, or **Windows**. 
 
-To test that you have Swift installed, run `swift --version` from your shell or terminal app.
+> Tip: To test that you have Swift installed, run `swift --version` from your shell or terminal app.
 
-Swift comes bundled with the [Swift Package Manager (SwiftPM)]({% link documentation/package-manager/index.md %}), which manages the distribution of your Swift code, and allows easy importing of other Swift packages into your apps.
+Swift comes bundled with the [Swift Package Manager (SwiftPM)]({% link documentation/package-manager/index.md %}), which manages the distribution of your Swift code and allows easy importing of other Swift packages into your apps. SwiftPM is also used for managing dependencies in Swift server-side projects, command-line tools, and Swift scripts.
```
Comment (federicobucchi) [inline _includes/getting-started/_installing.md:7]: Personally this seems redundant:

> SwiftPM is also used for managing dependencies in Swift server-side projects, command-line tools, and Swift scripts.

**Review by federicobucchi — APPROVED** [review]
Just a commend and LGTM

File: _includes/getting-started/_installing.md (hunk 1 above)
Comment (shahmishal) [reply _includes/getting-started/_installing.md:7]: "Swift scripts" does not support SwiftPM today.

Comment (shahmishal) [reply _includes/getting-started/_installing.md:7]: cc: @bnbarham for SwiftPM review

Comment (RobinBateman808) [reply _includes/getting-started/_installing.md:7]: Ack, removing Swift scripts language.

Comment (RobinBateman808) [reply _includes/getting-started/_installing.md]: removed redundancy.

Comment (RobinBateman808) [reply _includes/getting-started/_installing.md]: @shahmishal - do you have further suggestions?

File: _includes/getting-started/_installing.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ## Installing Swift
 
-If you don't have Swift installed, [install it first](/install).
+To kickstart your journey, [install Swift](/install) to begin using it on **macOS**, **Linux**, or **Windows**. 
 
-To test that you have Swift installed, run `swift --version` from your shell or terminal app.
+> Tip: To test that you have Swift installed, run `swift --version` from your shell or terminal app.
 
-Swift comes bundled with the [Swift Package Manager (SwiftPM)]({% link documentation/package-manager/index.md %}), which manages the distribution of your Swift code, and allows easy importing of other Swift packages into your apps.
+Swift comes bundled with the [Swift Package Manager (SwiftPM)]({% link documentation/package-manager/index.md %}) that manages the distribution of your Swift code. It allows easy importing of other Swift packages into your apps, making it a valuable tool for any Swift developer.
```
Comment (0xTim) [inline _includes/getting-started/_installing.md:7]: Nit
```suggestion
Swift comes bundled with the [Swift Package Manager (SwiftPM)]({% link documentation/package-manager/index.md %}) that manages the distribution of Swift code. It allows easy importing of other Swift packages into your applications and libraries, making it a valuable tool for any Swift developer.
```

Comment (RobinBateman808) [reply _includes/getting-started/_installing.md:7]: @0xTim - great clarification! Thanks!

File: _includes/getting-started/_installing.md (hunk 1 above)
Comment (RobinBateman808) [reply _includes/getting-started/_installing.md]: @shahmishal - friendly reminder to review. Thank you! :-)

**Review by bnbarham — APPROVED** [review]

**Comment by 0xTim** [comment]
@swift-ci please test

**Pull request merged**